fbrp/src/fbrp/runtime/conda.py

```diff
@@ -108,7 +108,9 @@ async def run_cmd_with_envvar(self, conda_env):
             executable="/bin/bash",
             cwd=self.proc_def.root,
             env=conda_env,
+            preexec_fn=os.setsid,
```

Comment:
start_new_session=True is preferred over preexec_fn

Reply:
Done